reconcile/cna/client.py

```diff
@@ -23,15 +70,44 @@ def list_assets(self) -> list[dict[str, Any]]:
         cnas = self._ocm_client.get(api_path="/api/cna-management/v1/cnas")
         return cnas.get("items", [])
 
+    def fetch_bindings_for_asset(self, asset: Asset) -> list[dict[str, str]]:
+        """
+        Currently bindings can only be retrieved per asset.
+        I.e., we will need one GET call per asset to aquire
+        all bindings.
+        """
+        bindings = self._ocm_client.get(api_path=f"{asset.href}/bind")
+        return bindings.get("items", [])
+
     def create(self, asset: Asset, dry_run: bool = False):
         if dry_run:
-            logging.info("CREATE %s", asset)
+            logging.info(
+                "CREATE %s %s %s",
+                asset.asset_type().value,
+                asset.name,
+                asset.raw_asset_parameters(True),
+            )
             return
         self._ocm_client.post(
             api_path="/api/cna-management/v1/cnas",
             data=asset.api_payload(),
         )
 
+    def bind(self, asset: Asset, dry_run: bool = False):
```

Comment:
I wonder whether the client should care about `dry_run` at all? I'm not sure how much value it is in adding the logging statement. The caller would already know that it called `bind` and could handle this further up the chain?

Reply:
i've removed the log statement but kept the dry_run parameter. i still have high hopes that a `dry_run` for bind will be provided, at least some basic validation for it like the existence of the referenced cluster and maybe even the namespace

Reply:
That's fine with me.